zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/multitenancy/TenantAwareUpdateVariablesTest.java

```diff
@@ -63,41 +62,6 @@ public void shouldUpdateVariablesForDefaultTenant() {
         .hasIntent(VariableDocumentIntent.UPDATED);
   }
 
-  @Test
-  public void shouldRejectUpdateVariablesForUnauthorizedTenant() {
```

Comment:
❌ Same as the other tests. This is still a valid case we should be covering.

Reply:
Same problem with all the tenantAwareTest, the authorization is not enabled